src/workloads/execution/MatchFilters.yml

```diff
@@ -0,0 +1,342 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/query"
+Description: >
+  This workload tests a set of match filters.
+
+Keywords:
+  - Loader
+  - CrudActor
+  - QuiesceActor
+  - insert
+  - find
+
+GlobalDefaults:
+  Database: &Database test
+  Collection: &Collection Collection0
+  DocumentCount: &DocumentCount 1e6
+  Repeat: &Repeat 10
+  Threads: &Threads 1
+  MaxPhases: &MaxPhases 16
+
+Actors:
+# Phase 0: Insert documents into the collection.
+- Name: InsertData
+  Type: Loader
+  Threads: 1
+  Phases:
+    OnlyActiveInPhases:
+      Active: [0]
+      NopInPhasesUpTo: *MaxPhases
+      PhaseConfig:
+        Repeat: 1
+        Database: *Database
+        Threads: 1
+        CollectionCount: 1
+        DocumentCount: *DocumentCount
+        BatchSize: 1000
+        Document:
+          # NOTE: These documents will not produce a match for any of the filters below, forcing
+          # each query to scan the entire collection.
+          a: 0
+          simple_arr: [1,2,3,4,5,6,7,8,9,10]
+          arr_with_object_elements: [{b: 0, c: 0}, {b: 0, c: 0}, {b: 0, c: 0},{b: 0, c: 0},{b: 0,c: 0},{b: 0, c: 0}, {b: 0, c: 0},{b: 0, c: 0},{b: 0, c: 0},{b: 0, c: 0}]
```

Comment:
[question] Is there an expected format for these yaml files (as in, do these need to be clang formatted in the same way as the main mongo repo?)

Similarly, do I need a new line at the bottom of the file?

Reply:
You can run `./run-genny lint-yaml` to see if the yaml files adhere to the defined format.

Reply:
Done; thanks for the tip